sources/packages/api/src/services/institution/institution.service.ts

```diff
@@ -37,7 +37,7 @@ import {
 import { AccountDetails } from "../bceid/account-details.model";
 import { InstitutionUserAuthDto } from "../../route-controllers/institution/models/institution-user-auth.dto";
 import {
-  databaseFieldOfUserDataTable,
+  sortUsersSummary,
```

Comment:
I would say the "Summary" seems to be UI specific. I would recommend something like _sortUsersColumnMap_.